tests/utils/0030_df.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Verify zenfs df command works as expected.
+
+$ZENFS_DIR/zenfs df --zbd=$ZDEV >> $TEST_OUT
+RES=$?
+if [ $RES -ne 0 ]; then
+  exit $RES
+fi
+
+LINES=$(cat $TEST_OUT | wc -l)
+if [ $LINES -ne 4 ]; then
```

Comment:
This just checks that the output is four lines, can't we come up with something that would provide better test coverage? Actually sanity checking the numbers?

Reply:
Removed it. Check only if the df subcommand is run successfully.
Also, for the small workload that it runs, the space amp is 0 and does not make sense to check it.